packages/ckeditor5-ui/src/toolbar/toolbarview.js

```diff
@@ -216,6 +217,16 @@ export default class ToolbarView extends View {
 		 * @member {module:ui/toolbar/toolbarview~ToolbarBehavior}
 		 */
 		this._behavior = this.options.shouldGroupWhenFull ? new DynamicGrouping( this ) : new StaticLayout( this );
+
+		if ( this.options.shouldGroupWhenFull && this.options.isFloating ) {
+			this.extendTemplate( {
```

Comment:
Do we need this `extendTemplate`? I mean, technically this is correct because the method will do the job but it will consume some time, and generally speaking, it was designed to extend templates of other views (when you don't have access to the original template definition because it is a parent class etc.).

This could be replaced by:

```
const classes = [
    'ck',
    'ck-toolbar',
    bind.to( 'class' ),
    bind.if( 'isCompact', 'ck-toolbar_compact' )
];

if ( this.options.shouldGroupWhenFull && this.options.isFloating ) {
    classes.push( 'ck-toolbar_floating' );
}


this.setTemplate( {
    tag: 'div',
    attributes: {
        classes,
        role: 'toolbar',
        'aria-label': bind.to( 'ariaLabel' ),
        style: {
            maxWidth: bind.to( 'maxWidth' )
        }
    },

    children: this.children,

    on: {
        // https://github.com/ckeditor/ckeditor5-ui/issues/206
        mousedown: preventDefault( this )
    }
} );
```